components/apimgt/org.wso2.carbon.apimgt.rest.api.common/src/main/resources/publisher-api.yaml

```diff
@@ -9993,6 +10178,23 @@ paths:
 
 components:
   schemas:
+    ContentPublishStatus:
```

Comment:
Add example values to the properties and that will generate samples in try out

Reply:
Addressed by [8th set of required changes](https://github.com/wso2/carbon-apimgt/pull/12874/commits/3eefad006cdd2122381e6ff1706f1e9650d1b938)